doc/personalize.rst

```diff
@@ -22,3 +22,42 @@ Your images must have one of the following extensions:
 * .svg
 
 When you reload your RAMP page the new ``Powered by`` section should appear.
+
+
+Privacy Policy page
+-------------------
+
+You can add an optional Privacy Policy page, setting the following in the main
+`config.yaml`,
+
+.. code::
+
+    flask:
+        ...
+        privacy_policy_page: "<path.html>"
+
+Where the ``privacy_policy_page`` can be either a path to an HTML file, or
+directly the HTML contents of that page.
+
+This will enable the ``/privacy_policy`` endpoint, and will add it to the footer
+menu.
+
+
+Sign up and login pages
+-----------------------
+
+You can add personalized messages to the Sign Up and Login pages, as follows,
```

Comment:
maybe for this section good to repeat where it has to be added (main
`config.yaml`)?